src/components/node-list/node-list.test.js

```diff
@@ -9,7 +10,12 @@ describe('NodeList', () => {
     const search = wrapper.find('.pipeline-node-list-search');
     const nodeList = wrapper.find('.pipeline-node-list');
     expect(search.length).toBe(1);
-    expect(nodeList.length).toBe(1);
+    expect(nodeList.length).toBeGreaterThan(0);
+  });
+
+  it('renders null if hasData is falsey', () => {
+    const wrapper = setup.mount(<NodeList />, loadData([]));
+    expect(wrapper.html()).toBeFalsy();
```

Comment:
What happens if loadData is supplied with undefined or null? (same expect or throws error)

Reply:
Throws error

Reply:
👍